pkgs/development/libraries/hsa-amd-aqlprofile/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, dpkg
+, hip
+}:
+
+# This is currently closed-source...
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hsa-amd-aqlprofile";
+  rocmVersion = "5.3.3";
+  modVersion = with lib.strings; concatStrings (intersperse "0" (splitString "." finalAttrs.rocmVersion));
+  magicVersion = "99";
+  ubuntuVersion = "22.04";
+  version = finalAttrs.rocmVersion;
+
+  src = fetchurl {
+    url = "https://repo.radeon.com/rocm/apt/${finalAttrs.rocmVersion}/pool/main/h/${finalAttrs.pname}/${finalAttrs.pname}_1.0.0.${finalAttrs.modVersion}-${finalAttrs.magicVersion}~${finalAttrs.ubuntuVersion}_amd64.deb";
```

Comment:
Would recommend to use a with finalAttrs for this one line to reduce code duplication